src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final int indexRandom = new Random().nextInt(Colors.values().length);
```

Comment:
move indexRandom to getRandomColor method, and rename indexRandom to randomIndex or index
```suggestion
    private final int indexRandom = new Random().nextInt(Colors.values().length);
```